src/Tribe/Ticket_Object.php

```diff
@@ -662,7 +691,19 @@ public function remaining() {
 		 * @return int
 		 */
 		public function available() {
-			// if we aren't tracking stock, then always assume it is in stock or capacity is unlimited
+			// if we aren't tracking stock, then always assume it is in stock or capacity is unlimited.
+
+			/* @var Tribe__Cache $cache */
```

Comment:
:warning: This comment is 50% valid code; is this commented out code?

<sub>hash: 21c5cf9f0d3aaea09954925f630abcc0</sub>